### DISCUSSION TITLE
Laravel-Vue in vercel

### DISCUSSION THREAD

**Discussion by ako-precious** [vercel/community#4041] [Help] [answered]
### Summary

After uploading the Laravel-Vue application to Vercel. I got these errors with means it cant find my application files. 

GET https://akos-portfolio.vercel.app/build/assets/app-eabffe08.js net::ERR_ABORTED 404
akos-portfolio.vercel.app/:25 
        
        
        GET https://akos-portfolio.vercel.app/build/assets/app-e7551d1f.css net::ERR_ABORTED 404
akos-portfolio.vercel.app/:1 
        
        
        GET https://akos-portfolio.vercel.app/build/assets/_commonjsHelpers-de833af9.js net::ERR_ABORTED 404
akos-portfolio.vercel.app/:1 
        
        
        GET https://akos-portfolio.vercel.app/build/assets/scrollbar-06ca61c4.js net::ERR_ABORTED 404
logo.jpg:1 
        
        
        GET https://akos-portfolio.vercel.app/logo.jpg 404
Image (async)
(anonymous) @ inpage.js:1
u @ inpage.js:1
a @ inpage.js:1
(anonymous) @ inpage.js:1
r.sendSiteMetadata @ inpage.js:1
e @ inpage.js:1
logo.jpg:1 
        
        
        GET https://akos-portfolio.vercel.app/logo.jpg 404
Image (async)
(anonymous) @ inpage.js:1
u @ inpage.js:1
a @ inpage.js:1
await in a (async)
(anonymous) @ inpage.js:1
r.sendSiteMetadata @ inpage.js:1
e @ inpage.js:1
logo.jpg:1 
        
        
        GET https://akos-portfolio.vercel.app/logo.jpg 404
Image (async)
(anonymous) @ inpage.js:1
u @ inpage.js:1
a @ inpage.js:1
await in a (async)
(anonymous) @ inpage.js:1
r.sendSiteMetadata @ inpage.js:1
e @ inpage.js:1
akos-portfolio.vercel.app/:1  Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.
VM299:1  Uncaught DOMException: Failed to execute 'appendChild' on 'Node': The new child element contains the parent.
    at saveOptions_s (<anonymous>:1:44854)
    at getMessages_s (<anonymous>:1:39459)

### Example

https://akos-portfolio.vercel.app/

### Steps to Reproduce

This is my vercel.json file

{
   
    "version": 2,
    
    "framework": "vite",

    "functions":{
        "api/index.php": { "runtime": "vercel-php@0.6.0"}
    },
    "routes": [
        
        {
            "src": "/(.*)",
            "dest": "/api/index.php"
        }
    ],
   
    "env": {
        "APP_ENV": "production",
        "APP_DEBUG": "true",
        "APP_URL": "https://yourproductionurl.com",

        "APP_CONFIG_CACHE": "/tmp/config.php",
        "APP_EVENTS_CACHE": "/tmp/events.php",
        "APP_PACKAGES_CACHE": "/tmp/packages.php",
        "APP_ROUTES_CACHE": "/tmp/routes.php",
        "APP_SERVICES_CACHE": "/tmp/services.php",
        "VIEW_COMPILED_PATH": "/tmp",

        "CACHE_DRIVER": "array",
        "LOG_CHANNEL": "stderr",
        "SESSION_DRIVER": "cookie"
    }
}

**Comment by ako-precious** [comment, marked as answer]
The solution is here https://github.com/orgs/vercel/discussions/1296

**Reply by amyegan** [reply]
Thanks for linking the answer here! 🙇
